### ISSUE TITLE
Checkra1n crashing after entering DFU Mode

### ISSUE THREAD

**Issue by yenaldo** [checkra1n/BugTracker#2060]
**Tell us about your setup:**
1. What iDevice are you using?
- iPhone SE
2. On what version of iOS is it?
14.4
3. What version of checkra1n are you using?
0.12.0
4. What is your host system (OS version? Hackintosh? VM? etc.)?
Mac OSX Mojave 10.14.6
5. How are you connecting to the device (USB-A? USB-C? Apple/3rd party cable? Through a USB hub?)?
Directly with an original USB-Cable

**What are the steps to reproduce the issue?**
1. Checkra1n opens up
2. I follow the instructions
3. After entering the DFU Mode iTunes appears and checkra1n crashes
...

**What do you expect, and what is happening instead?**
I've tried an other iPhone, same issue.

**Did you see a popup on the device stating it entered "Safe Mode" due to an error?**
No.

**Does the issue also occur if you tick "Safe Mode" in the checkra1n options?**
Can't enter safe mode.

**Any other info, error logs, screenshots, ...?**
No.

**Comment by brandonplank** [comment]
Thumbs up to this Exact error "Seg 11"

Using -c


```
#
# Checkra1n beta 0.12.2
#
# Proudly written in nano
# (c) 2019-2020 Kim Jong Cracks
#
#========  Made by  =======
# argp, axi0mx, danyl931, jaywalker, kirb, littlelailo, nitoTV
# never_released, nullpixel, pimskeks, qwertyoruiop, sbingner, siguza
#======== Thanks to =======
# haifisch, jndok, jonseals, xerub, lilstevie, psychotea, sferrini
# Cellebrite (ih8sn0w, cjori, ronyrus et al.)
#==========================

 - [*]: Waiting for DFU devices
 - [*]: Exploiting
 - [*]: Checking if device is ready
zsh: segmentation fault  /Applications/checkra1n.app/Contents/MacOS/checkra1n -c -k 
```

**Comment by brandonplank** [comment]
[checkra1n_2021-04-27-142639_Brandons-MacBook-Pro.crash.zip](https://github.com/checkra1n/BugTracker/files/6386761/checkra1n_2021-04-27-142639_Brandons-MacBook-Pro.crash.zip)

Crash log.

**Comment by yenaldo** [comment]
What are you mean?

**Comment by ghostbandana** [comment]
I’m having the same issue. As soon as chechra1n starts to inject the program crashes. I’m on iPhone X iOS 14.4.2

**Comment by heyhello1** [comment]
I have the same issue

**Comment by yenaldo** [comment]
Anyone a solution? Please help guys! :(

**Comment by ghostbandana** [comment]
[SOLVED] If you open settings and go to upgrade firmware and it says error, you have to backup to your computer and restore your device from iTunes. This is the only way to fix segmentation fault 11.

**Comment by verylovestars** [comment]
Same trouble

**Comment by Siguza** [comment]
Should be fixed in 0.12.3.
